requirements.txt

```diff
@@ -1,19 +1,18 @@
 nbmake==1.4.1
 astropy==5.1.1
 Cython==0.29.32
-h5py==3.7.0
+h5py==3.9.0
 mpmath==1.3.0
 numpy==1.23.4
 packaging==21.3
 pyerfa==2.0.0.1
 pyparsing==3.0.9
 PyYAML==6.0
 scipy==1.9.3
-sympy==1.11.1
 unyt==2.9.2
 cmasher==1.6.3
-numba==0.56.4
 git+https://github.com/karllark/dust_attenuation.git@v0.5.dev
 dust_extinction==1.1
 nbsphinx==0.9.1
 matplotlib==3.7.1
+spectres==2.2.0
```

Comment:
What is spectres and why are we changing h5py version?

Reply:
spectres is a package for re-binning spectra and its used by `rebin_grid.py`. The older `h5py` version was breaking things for me.